pramen/core/src/main/scala/za/co/absa/pramen/core/runner/task/TaskRunnerMultithreaded.scala

```diff
@@ -83,16 +83,19 @@ class TaskRunnerMultithreaded(conf: Config,
   }
 
   override def runSequential(tasks: Seq[Task]): Future[Seq[RunStatus]] = {
-    Future {
-      val requiredResources = tasks.map(_.job.operation.consumeThreads).max
-      whenEnoughResourcesAreAvailable(requiredResources) {
-        runDependentTasks(tasks)
+    if (tasks.isEmpty) {
```

Comment:
This is the fix - the task list can be empty if the underlying job is not daily and the pipeline is daily.